addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -75,7 +75,6 @@ $radio-svg: (
   }
   .rose-form-radio-label {
     @include type.type(s, bold);
-    display: flex;
```

Comment:
It looks like this `display` was always getting overriden by another `display: block` on line 82